public/data/ShlokMohanty.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Shlok Mohanty",
+  "bio": "Member of EddieHub, Machine Learning ,Data Science , Full stack Development enthusiast",
+  "avatar": "https://github.com/ShlokMohanty.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/ShlokMohanty",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/mohanty_shlok",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Facebook",
+      "url": "https://www.facebook.com/shlok.mohanty.35",
+      "icon": "facebook"
+    }
+  ]
+}
+  
```

Comment:
There's an extra line here that's not needed.
**NOTE:** there are 2 extra lines, as the Git diff doesn't display the endline.
```suggestion
```

Reply:
@Panquesito7 so i should delete this line?

Reply:
@Panquesito7 now its fine ?